CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Change Log
+All notable changes to this project will be documented in this file.
+This project adheres to [Semantic Versioning](http://semver.org/).
+
+## [1.1.0] - 2024-04-03
+## Update
+- `life` as a valid top-level domain for email format validation
+- bump mocha to ^10.4.0; update npm test script accordingly
+
+## [1.0.1] - 2024-03-11
+## Bump
+- uglify-js to ^2.6
+
+## [1.0.0] - 2024-03-11
+## Fork
+- initial fork of themis@v1.1.6
+- no updates in 9 years from publisher
```

Comment:
I do appreciate this comment, but I'm not sold on maintaining a changelog. As demonstrated by this PR, they are easy to forget, and hard to ensure they get updated. Moreover, we don't use them consistently (they are common (but not universal) in our libs, but not used at all in our API and client repos 🤷🏻 why the discrepancy?)

Personally I'd lean towards documenting the "why we forked it" in the readme, and skipping the changelog. I think the git history typically has a very similar summary of changes to the changelog

Reply:
Yeah that's fair. We can remove in another PR if we want to.